### ISSUE TITLE
Increased tyrosine in urine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6047]
**Preferred term label:** Increased tyrosine in urine

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of tyrosine in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Neutral hyperaminoaciduria HP:0008353

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding as HP:0033091
